recipes/natural_language_processing/chatbot/Makefile

```diff
@@ -24,7 +24,7 @@ quadlet:
 	    -e "s|APPIMAGE|${APPIMAGE}|g" \
 	    -e "s|MODELIMAGE|${MODELIMAGE}|g" \
 	    quadlet/${APP}.yaml \
-	    > /tmp/${APP}.yaml
+	    > build/${APP}.yaml
```

Comment:
Maybe `mkdir -p .build` instead (and change L22,L27 to `.build/${APP}.*`) ? and then in .gitignore add this:
*.build*

Reply:
.build would make it harder for users to find?  Why obscure this?